wamr-compiler/main.c

```diff
@@ -632,6 +634,15 @@ main(int argc, char *argv[])
         goto fail2;
     }
 
+    /* Only if SIMD is actually used, we need to set the compiler
+     * options */
+    if (((WASMModule *)wasm_module)->is_simd_used) {
+        option.enable_simd = true;
+    }
+    else {
+        option.enable_simd = false;
+    }
```

Comment:
Is it better to set the flag inside aot_create_comp_data? The `wasm_module` is passed into the function.

Reply:
If that, handle it in aot_create_comp_context seems better.

Reply:
Yes, maybe it is better to handle it in aot_create_comp_context.